src/presentation/ui/page/databases.templ

```diff
@@ -0,0 +1,299 @@
+			},
+
+			// Auxiliary states
+			changeSelectedDatabaseType(databaseType) {
+				htmx.ajax(
+					'GET',
+					'/databases/?dbType=' + databaseType,
+					{
+						select: '#databases-content',
+						target: '#databases-content',
+						swap: 'outerHTML transition:true'
+					},
+				);
+			},
+			installDatabaseService(databaseType) {
+				htmx.ajax(
+					'POST',
+					'/api/v1/services/installables/',
+					{
+						swap: 'none',
+						values: { name: databaseType },
+					},
+				);
+			},
+			get shouldDisableCreateDatabaseSubmitButton() {
+				return this.database.name == ''
+			},
+			get shouldDisableCreateDatabaseUserSubmitButton() {
+				return this.database.name == '' || this.databaseUser.username == '' || this.databaseUser.password == ''
+			},
+
+			// Modal states
+			isCreateDatabaseModalOpen: false,
+			openCreateDatabaseModal() {
+				this.resetPrimaryStates();
+
+				this.isCreateDatabaseModalOpen = true;
+			},
+			closeCreateDatabaseModal() {
+				this.isCreateDatabaseModalOpen = false;
+			},
+			isDeleteDatabaseModalOpen: false,
+			openDeleteDatabaseModal(databaseName) {
+				this.resetPrimaryStates();
+
+				this.database.name = databaseName;
+				this.isDeleteDatabaseModalOpen = true;
+			},
+			closeDeleteDatabaseModal() {
+				this.isDeleteDatabaseModalOpen = false;
+			},
+			deleteDatabaseElement(databaseType) {
+				htmx.ajax(
+					'DELETE',
+					'/api/v1/database/' + databaseType + '/' + this.database.name + '/',
+					{swap: 'none'},
+				);
+				this.closeDeleteDatabaseModal();
+			},
+			isCreateDatabaseUserModalOpen: false,
+			openCreateDatabaseUserModal() {
+				this.resetPrimaryStates();
+
+				this.isCreateDatabaseUserModalOpen = true;
+			},
+			closeCreateDatabaseUserModal() {
+				this.isCreateDatabaseUserModalOpen = false;
+			},
+			createDatabaseUser(databaseType) {
+				htmx.ajax(
+					'POST',
+					'/api/v1/database/' + databaseType + '/' + this.database.name + '/user/',
+					{
+						swap: 'none',
+						values: {
+							username: this.databaseUser.username,
+							password: this.databaseUser.password,
+							privileges: this.databaseUser.privileges,
+						}
+					},
+				);
+				this.closeCreateDatabaseUserModal();
+			},
+			isDeleteDatabaseUserModalOpen: false,
+			openDeleteDatabaseUserModal(databaseName, databaseUsername) {
+				this.resetPrimaryStates();
+
+				this.database.name = databaseName;
+				this.databaseUser.username = databaseUsername;
+				this.isDeleteDatabaseUserModalOpen = true;
+			},
+			closeDeleteDatabaseUserModal() {
+				this.isDeleteDatabaseUserModalOpen = false;
+			},
+			deleteDatabaseUserElement(databaseType) {
+				htmx.ajax(
+					'DELETE',
+					'/api/v1/database/' + databaseType + '/' + this.database.name + '/user/' + this.databaseUser.username + '/',
+					{swap: 'none'},
+				);
+				this.closeDeleteDatabaseUserModal();
+			},
+		}))
+	})
+}
+
+templ DatabasesIndex(selectedDbTypeSummary presenterDto.SelectedDatabaseTypeSummary) {
```

Comment:
What is a Summary? I have no idea what this means here.

Reply:
Changed to `DatabaseOverview`!